dists/emscripten/build.sh

```diff
@@ -0,0 +1,207 @@
+#!/bin/bash
+# print commands
+set -o xtrace
+# exit when any command fails
+set -e
+if [ "$#" -ne 1 ]; then
+  echo "$0: exactly 1 arguments expected: configure, make, data, all"
+  exit 3
+fi
+
+EMSDK_VERSION="2.0.23"
+ROOT_FOLDER=$(pwd)
+DIST_FOLDER="$ROOT_FOLDER/dists/emscripten"
+LIBS_FOLDER="$DIST_FOLDER/libs"
+if [[ ! -d "$DIST_FOLDER" ]]; then
+  echo "/dists/emscripten/ not found. Please make sure to run this script from the root of the project - ./dists/emscripten/build.sh "
+  exit 1
+fi
+if [[ "$1" =~ ^(clean)$ ]]; then
+  make clean
+  rm -rf ./dists/emscripten/libs/build
+  rm -rf ./dists/emscripten/libs/*/
+  rm -rf ./dists/emscripten/emsdk*/
+  find . -name "*.o"
```

Comment:
This has a side effect of removing gph/gp2x/mmuhack/mmuhack.o (which reminds me that I need to clean that port up). Why `make distclean` is not working for you?

Reply:
You are right. This was overaggressive. I changed it to only remove a single file that isn't taken care of by `make distclean`.